README.md

```diff
@@ -63,7 +63,7 @@ https://snapcraft.io/hamster-snap
 ###### Ubuntu (tested in 19.04 and 18.04)
 
 ```bash
-sudo apt install gettext intltool python3-gi-cairo python3-distutils python3-dbus python3-xdg
+sudo apt install gettext intltool python3-gi-cairo python3-distutils python3-dbus python3-xdg libglib2.0-dev-bin
```

Comment:
```suggestion
sudo apt install gettext intltool python3-gi-cairo python3-distutils python3-dbus python3-xdg libglib2.0-dev
```
From `apt info libglib2.0-dev-bin`:
>  This package is needed to compile programs against libglib2.0-0. It contains
 development utilities typically run during compilation and should not be
 installed directly. Use libglib2.0-dev instead.

Reply:
Installing libglib2.0-dev instead of just libglib2.0-dev-bin causes a butt load of other dependency packages being installed without a need for them. Which is a bit of a shame.

It makes me wonder how much of a need we have for glib-genmarshal to begin with. I suppose we can't just drop the requirement?
If not than yes, better to stick to the recommendation to prevent issues for the user later on.

Reply:
> It makes me wonder how much of a need we have for glib-genmarshal to begin with. I suppose we can't just drop the requirement?

Good questions. If anyone finds evidence that's it's not required 
and ways to silence the complaint, 
that would be great, maybe after v3.0 as time is short.

> If not than yes, better to stick to the recommendation to prevent issues for the user later on.

Agreed, thanks again !

Reply:
Looks like I cannot push this recommendation myself
(there was a checkbox to allow edits by maintainer to allow that).
It's OK, but it's up to you now.